the-new-octal/src/app/page/products/products.component.html

```diff
@@ -53,6 +53,16 @@ <h4 class="card-title mt-0">Products</h4>
                       </button>
                     </td>
                   </tr>
+                  <tr>
+                    <td class="table-active font-weight-bold text-danger h3">Összesen: </td>
+                    <td class="table-active font-weight-bold h4">{{sumName$ | async}}</td>
+                    <td class="table-active font-weight-bold h4">{{sumType$ | async}}</td>
+                    <td class="table-active font-weight-bold h4"></td>
+                    <td class="table-active font-weight-bold h4">{{sumDescription$ | async}}</td>
+                    <td class="table-active font-weight-bold h4">{{sumPrice$ | async}}</td>
+                    <td class="table-active font-weight-bold h4">{{sumFeatured$ | async}}</td>
+                    <td class="table-active font-weight-bold h4">{{sumActive$ | async}}</td>
+                  </tr>
```

Comment:
Az a piros összesen szó szerintem nem illik a design-hoz, az összes oszlophoz tartozzon td.

Reply:
Ok, a színt leveszem, a td-ket üresen is berakom.